### DISCUSSION TITLE
Calculate correct transform

### DISCUSSION THREAD

**Discussion by codeninjacoaching** [xyflow/xyflow#3587] [Q&A]
Amazing work guys. I have a quick question. I am trying to use this on a mobile but there is a lot of gap between the first node and the the top. How can I fix it please? Here is the code I have used.
https://codesandbox.io/embed/pedantic-sea-fnktt4?codemirror=1
I have also attached a [screenshot[](url)](<img width="441" alt="Screenshot 2023-11-08 at 18 12 42" src="https://github.com/xyflow/xyflow/assets/128642672/3d9404f9-500b-401b-b502-b32d77769fcd">)

**Comment by moklick** [comment]
That's the expected behaviour when you use `fitView`. Your flow has a different aspect ratio than the screen and fitView centers it.

**Reply by codeninjacoaching** [reply]
Thanks @moklick . Is there a way to align it to the top somehow based on the screen size? I can use a media query?
Any idea how can I achieve this? or any other alternate?
